### DISCUSSION TITLE
CookieCloud同步失败：cookie解密失败

### DISCUSSION THREAD

**Discussion by HomeQi** [jxxghp/MoviePilot#2654] [问题咨询]
第一次见到这个问题，网上没有相关的回答，尝试修改脚本也无效。。。希望大佬能看一下什么原因。
【ERROR】2024-08-23 09:29:40,813 - site.py - CookieCloud同步失败：cookie解密失败：'utf-8' codec can't decode byte 0xe0 in position 0: invalid continuation byte
【INFO】2024-08-23 09:29:40,803 - site.py - 开始同步CookieCloud站点 ...

**Comment by gken6** [comment]
我也碰到这个问题了，请问你有解决了吗？

**Reply by HomeQi** [reply]
清除，重新构建一下容器

**Comment by gken6** [comment]
已解决，重新生成了下账号密钥。谢谢回复

HomeQi ***@***.***> 于2024年10月31日周四 19:46写道：

**Reply by no2coolboy** [reply]
是cookiecloud重新生成uuid和密码吗

**Comment by gken6** [comment]
是的

no2coolboy ***@***.***> 于2025年11月7日周五 10:14写道：
